src/t_zset.c

```diff
@@ -142,11 +167,9 @@ zskiplist *zslCreate(void) {
     return zsl;
 }
 
-/* Free the specified skiplist node. The referenced SDS string representation
- * of the element is freed too, unless node->ele is set to NULL before calling
- * this function. */
+/* Free the specified skiplist node. No need to explicitly free the SDS string 'ele',
+ * as it is embedded within the structure. */
```

Comment:
(regarding zslFreeNode) The comment above the function is API documentation for the caller of the function. No need to explain how it works internally.

That the element is embedded in the same allocation is implementation details. If you think it's important to mention it somewhere, we can add a comment about it inside the body of the function instead.

```suggestion
/* Free the specified skiplist node. */
```